integrated_channels/integrated_channel/apps.py

```diff
@@ -1,7 +1,7 @@
 """
 Enterprise Integrated Channel Django application initialization.
 """
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
```

Comment:
Would be helpful to include a comment describing why absolute_import is necessary -- maybe also include link to the Django documentation if pertinent